.CI/common.groovy

```diff
@@ -178,6 +178,24 @@ void buildOMC(CC, CXX, extraFlags) {
      echo make -f 'Makefile.omdev.mingw' \${MAKETHREADS} BUILDTYPE=Release runtimeCPPmsvcinstall
      echo cd \${MSYS_WORKSPACE}
      echo make -f 'Makefile.omdev.mingw' \${MAKETHREADS} BUILDTYPE=Release runtimeCPPinstall
+     echo echo Check that omc can be started and a model can be build for NF OF with runtimes C Cpp FMU
```

Comment:
Do you actually need `echo` twice here?

Reply:
That will make it echo when you run the script later